lab/lab2/js/part3-application.js

```diff
@@ -39,17 +39,34 @@ var resetMap = function() {
   it down!
 ===================== */
 var getAndParseData = function() {
-  /* =====================
-    Fill out this function definition
-  ===================== */
+  var parsed_data = $.ajax("https://raw.githubusercontent.com/CPLN692-MUSA611/datasets/master/json/philadelphia-crime-snippet.json")
+  .done(function(ajaxResponseValue) {
+  myData = JSON.parse(ajaxResponseValue);
+  });
+  //return parsed_data;
 };
 
 /* =====================
   Call our plotData function. It should plot all the markers that meet our criteria (whatever that
   criteria happens to be — that's entirely up to you)
 ===================== */
 var plotData = function() {
-  /* =====================
-    Fill out this function definition
-  ===================== */
+  var return_list = [];//set criteria
+  _.each(myData, function(record) {
```

Comment:
I am able to plot a couple of points, so this seems to be a success, but can you help me understand what you are doing here? (just explain the goal with these filters)

Thanks.

Reply:
Sorry just saw this...here I'm selecting records that happened in District 1, and enable the user to choose the category and PSA between the maximum and minimum fields.
The console.log() s are just used to see if I filtered the records correctly.